script/docker/init.sh

```diff
@@ -5,9 +5,14 @@ set -ex
 # Change directory to root of the repo
 cd "$(dirname "$0")/../.."
 
-cp -b config/docker/database.yml config/database.yml
-cp -b config/docker/redis.yml config/redis.yml
-cp -b config/docker/local.yml config/local.yml
+copy="cp -b"
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    copy="cp" # macOS cp does not support the -b option
+fi
+
+$copy config/docker/database.yml config/database.yml
+$copy config/docker/redis.yml config/redis.yml
+$copy config/docker/local.yml config/local.yml
```

Comment:
Might be better to keep the same behavior across all versions by going verbose?
Something like:
```sh
for file in 'database.yml' 'redis.yml' 'local.yml'
do
  # Manual backup as the --backup option is not available for all versions of cp
  test -f "config/$file" && cp "config/$file" "config/$file~"
  cp "config/docker/$file" "config/$file"
done
```

Reply:
Yeah that makes sense! Done.